internal/wasi/fs_test.go

```diff
@@ -0,0 +1,21 @@
+package internalwasi
+
+import (
+	"testing"
+	"testing/iotest"
+
+	"github.com/stretchr/testify/require"
+)
+
+// TestMemFile_Read_Seek tests the behavior of Read and Seek by iotest.TestReader.
+// See iotest.TestReader
+func TestMemFile_Read_Seek(t *testing.T) {
```

Comment:
let me know if you know of the better naming

Reply:
fine with me!